src/devtools/useLocalStorage.ts

```diff
@@ -23,6 +28,7 @@ export default function useLocalStorage(key, defaultValue) {
     } else {
       setValue(initialValue)
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [defaultValue, key])
```

Comment:
what would be expected here from the linter please, and why do we have to disable it?

Reply:
It was the same dependency T issue, gone after upgrading `eslint-plugin-react-hooks`. Removed the comment ✅